package.json

```diff
@@ -107,7 +107,7 @@
     "react-pure-render": "1.0.2",
     "react-redux": "4.4.5",
     "react-tap-event-plugin": "2.0.1",
-    "react-virtualized": "7.9.1",
+    "react-virtualized": "8.8.1",
```

Comment:
We also need to update the `npm-shrinkwrap.json`. See [related documentation](https://github.com/Automattic/wp-calypso/blob/master/docs/shrinkwrap.md) for more information. Should be as simple as `make shrinkwrap` (after you've installed `npm -g shonkwrap`).